docs/motivation.md

```diff
@@ -36,8 +35,7 @@ const ValidatedQuickForm = ValidatedForm.Validated(QuickForm.Quick(BaseForm));
 ```
 
 <p align="center">
-  <img src="/img/uniforms-traits.png" />
-  <small style={{display: 'block'}}>AutoValidatedQuickSemanticForm (using 5 "traits") viewed in React DevTools.</small>
+  <img src="/img/uniforms-traits.png" alt='AutoValidatedQuickSemanticForm (using 5 "traits") viewed in React DevTools.' />
```

Comment:
Would be nice to mentioned that there is only one component rendered shown in the image.

Reply:
After consultation: "Single AutoValidatedQuickSemanticForm component (using 5 "traits") viewed in React DevTools."